Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
+RUN openssl genrsa 2048 > ssl-key.pem
+RUN openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem
+
+COPY . /src
+RUN cd /src; npm install
+
+WORKDIR /src/data
+RUN echo $'@prefix n0: <http://www.w3.org/ns/auth/acl#>. \n\
```

Comment:
OK, this is a teensy bit messy but it gives me an opportunity to demonstrate best practices for installing stuff so here goes:

The nicest way to manage this interpolation whilst also keeping everything in the file (in my opinion) is to use `envsubst`, which needs installing as part of the gettext package.

My test dockerfile for this looks like this:

```
FROM node:5.11-wheezy
MAINTAINER Kit Barnes <k.barnes@mhnltd.co.uk>

ARG blah

RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
    gettext \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*

COPY acl_template /src/data/acl_template
RUN cat /src/data/acl_template | envsubst > /src/data/.acl \
 && rm /src/data/acl_template
```